model/project_aliases.go

```diff
@@ -206,30 +206,6 @@ func FindAliasInProjectOrRepo(projectID, alias string) ([]ProjectAlias, error) {
 	if len(aliases) > 0 {
```

Comment:
can we change this to if len(aliases) > 0 || shouldExit rather than two statements?